01 - Community/People/javalent.md

```diff
@@ -1,25 +1,29 @@
 ---
 aliases:
 - Jeremy Valentine
+- Javalantasy
 tags:
-- 
+- incubator
 publish: true
 ---
 
-# Jeremy Valentine
+# Javalent
 
-- GitHub: [javalent](https://github.com/javalent/) ^github
-<!-- - Discord: `@` ^discord-->
-- Website: <https://github.com/valentine195> ^website
-<!-- - [[Publish sites|Publish site]]: <https://> ^publish-->
+- GitHub: [Javalent](https://github.com/javalent/) ^github
```

Comment:
please retain the case of the username.

Reply:
This is now fixed on my end and will be included in the next commit.